ui/components/app/user-preferenced-currency-display/user-preferenced-currency-display.stories.js

```diff
@@ -1,52 +1,27 @@
 import React from 'react';
-import { PRIMARY, SECONDARY, ETH } from '../../../helpers/constants/common';
+import { ETH } from '../../../helpers/constants/common';
 
 import UserPreferencedCurrencyDisplay from '.';
 
 export default {
   title: 'Components/App/UserPreferencedCurrencyDisplay',
   id: __filename,
   argTypes: {
-    className: {
```

Comment:
`className` is a prop that is accepted by the component.

Not entirely sure how to use it as a control though :thinking: Is there any "global" stylesheet that we could reference classes from? That might work.

Reply:
I check it by inspecting the html. I think that is an acceptable way to check?

Reply:
Oh of course. Just, it'd be nice to have some pre-made classes that are easier to test with. e.g. `background-red` to change the background. Something easy to input and simple to test. Just an idea.